src/main/java/io/ebean/bean/EntityBeanIntercept.java

```diff
@@ -859,7 +858,7 @@ public void initialisedMany(int propertyIndex) {
     loadedProps[propertyIndex] = true;
   }
 
-  private final void preGetterCallback() {
+  private void preGetterCallback() {
```

Comment:
Redundant. Making a method final only serves to prevent if from being override. Which making it private also does.

Explained here: https://stackoverflow.com/questions/23161362/why-can-private-method-be-final

EDIT: Dammit this wasn't supposed to be a review comment why is that the green option?